File: zkext/trunk/src/com/ff/ui/grid/Column.java
package com.ff.ui.grid;

import com.ff.annotation.Property;
import com.ff.ui.container.Container;

public class Column extends Container{
	
	@Property
	private String text;
	@Property
	private String name;

	public String getText() {
		return text;
	}

	public void setText(String text) {
		this.text = text;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	 

}

File: zkext/trunk/src/web/js/zkext/ui/grid/Column.js
/* Column.js*/

zkext.ui.grid.Column = zk.$extends(zkext.ui.container.Container,{	 
	$define: {	 
		 text:function(val){
			 this.setProperty("text","setText",val);
		 },
		 name:function(val){
			 val = val.replace(".",ESCAPE_FIELD);
			 this._name = val;
			 this.setProperty("dataIndex",null,val);
		 },
		 visible:function(val){
			 
		 }
	},	
	configure_:function(){	
		this.$supers('configure_',arguments);		
		var config = this.getInitialConfig();
		config.field = new Object();
		config.field.xtype = 'textfield';
	},
	createExt_:function(){
		this.newInstance("Ext.grid.column.Column");
	}
	 
});